gpkit/interactive/rendering.py

```diff
@@ -0,0 +1,35 @@
+"""Rendering methods"""
+
+"""Save a set of variables to a .csm, for usage
+with the Engineering Sketch Pad rendering software.
+
+Arguments:
+
+- filepath is a string with the full filepath to the .csm,
+for example "design.csm" or "renders/version1.csm"
+
+- sol is a GPkit Solution instance
+
+- paramdict is a dictionary comprised of GPkit Variables
+indexed by their design parameter name in ESP (String).
+"""
+
+def saveVariablesToCSM(filepath,sol,paramdict):
+        contents = ""
+        if filepath != "":
```

Comment:
if filepath is an optional argument, function signature should be `sol, paramdict, filepath=None`, and this conditional should be `if filepath is not None`